.plzconfig

```diff
@@ -1,5 +1,5 @@
 [Please]
-Version = >=17.2.0
+Version = >=17.10.0
```

Comment:
This change requires tri-state support for optional boolean plugin config fields, implemented in https://github.com/thought-machine/please/pull/3219.